azure-pipelines.yml

```diff
@@ -7,9 +7,9 @@ variables:
 jobs:
 - template: azure-templates.yml
   parameters:
-    name: Linux_37_offline
+    name: Linux_36_offline
```

Comment:
ooi, why did you switch the python versions over?

Reply:
It was to avoid the DB issue.

Reply:
So I switched it back to normal